app/main/helpers/buyers_helpers.py

```diff
@@ -93,7 +93,7 @@ def get_briefs_breadcrumbs(additional_breadcrumbs=[]):
         },
         {
             "link": url_for("buyers.buyer_dos_requirements"),
-            "label": "Your requirements"
+            "label": "Requirements"
```

Comment:
This change also breaks some unit tests

Reply:
Is this because the unit tests are checking the content or because changing the content has a knock-on effect on other parts of the app/service?

Reply:
It is because the unit tests are asserting what the breadcrumbs are.

The corresponding unit tests for this content can be found here:https://github.com/alphagov/digitalmarketplace-briefs-frontend/blob/9ed455b4e891253e31e4f2583d5868f5b9e7c094/tests/helpers.py#L263

You should just be able to make the identical content change to that line in the test file.

Let me (or one of the devs on your team) if this is unclear of you need any help :)

Reply:
This is something George picked up this morning. The tickets are going back into the backlog and we'll land these changes once the tests have been updated and passed too.
Thanks.

Reply:
Updated line 263 of the test file as part of this PR.

Reply:
@idavidmcdonald - is this okay now?